src/services/videoconference/logic/constants.js

```diff
@@ -36,6 +36,6 @@ exports.STATES = states;
 const createOptionToggles = {
 	MODERATOR_MUST_APPROVE_JOIN_REQUESTS: 'moderatorMustApproveJoinRequests',
 	EVERYBODY_JOINS_AS_MODERATOR: 'everybodyJoinsAsModerator',
-	EVERY_ATTENDY_JOINS_MUTED: 'everyAttendeJoinsMuted',
+	EVERY_ATTENDEE_JOINS_MUTED: 'everyAttendeJoinsMuted', // TODO: Fix string value "attendee" here and in client
```

Comment:
Please fix this value in client too

Reply:
I didn't dare to touch the literal value `'everyAttendeJoinsMuted'` as changing it would require changes to both the server and the client to be rolled out in sync (there would be separate PRs on 2 repos that depend on each other). What would be the intended process for this?

Reply:
but in another, next pr that we may merge this